docs/admin/cluster-management.md

```diff
@@ -80,6 +72,37 @@ If you want more control over the upgrading process, you may use the following w
     be created automatically when you create a new VM instance (if you're using a cloud provider that supports
     node discovery; currently this is only Google Compute Engine, not including CoreOS on Google Compute Engine using kube-register). See [Node](node.md).
 
+## Advanced Topics
+
+### Upgrading to a different API version
+When a new API is released, you may need to upgrade a cluster to support a new API version (e.g. switching from 'v1' to 'v2' when 'v2' is launched)
```

Comment:
When a new API is released -> When a new API version is released
support a new API version -> support the new API version